tests/components/tuya/fixtures/sp_csr2fqitalj5o0tq.json

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
Please add the fixture to `__init__.py`
It should generate snapshots for all platforms